README.md

```diff
@@ -183,3 +183,19 @@ __Runtime network setup with CNI__
 4. Create bridge, TAP, and link all together with network interface previously created ([code](https://github.com/containers/virtcontainers/blob/0.5.0/network.go#L123-L205))
 
 5. Start VM inside the netns and start the container ([code](https://github.com/containers/virtcontainers/blob/0.5.0/api.go#L66-L70))
+
+## Storage
+
+Container workloads are shared with the virtualized environment through 9pfs.
+The devicemapper storage driver is a special case. The driver uses dedicated block devices rather than formatted filesystems, and operates at the block level rather than file level. We use this knowledge to directly use the underlying block device instead of the overlay file system for the container root file system. The block device maps to the top read-write layer for the overlay. This approach gives much better I/O performance as compared to using 9pfs to share the container file system.
```

Comment:
s/rather than file level/rather than the file level/

s/performance as compared/performance compared/

Also, this change mixes familiar and passive voices. I'd personally reword to allow you to drop all the "We"'s.